azurerm/internal/services/network/data_source_network_service_tags.go

```diff
@@ -0,0 +1,105 @@
+package network
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/timeouts"
+)
+
+func dataSourceNetworkServiceTags() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceNetworkServiceTagsRead,
+
+		Timeouts: &schema.ResourceTimeout{
+			Read: schema.DefaultTimeout(5 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"location": azure.SchemaLocation(),
```

Comment:
Would this make more sense as `location_for_request` as it seems to only control which location to make the request to?

Reply:
I would rather specify it in the documentation because imho anyone who'd read `location_for_request` would anyway go to documentation. At least i would.